frontend/CLAUDE.md

```
@@ -96,6 +96,41 @@ npm run test:a11y:watch   # Watch accessibility tests during development
 - Component type mismatches
 - Accessibility violations
 
+## TypeScript Best Practices
+
+**CRITICAL: Never bypass TypeScript's type checking system**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)